### PR TITLE
DOCSP-23935 Disaster Recovery update

### PR DESCRIPTION
**Pull request by kennethdyer**
[DOCSP-23935](https://jira.mongodb.org/browse/DOCSP-23935)

[Staging](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-23935-disaster-recovery/reference/disaster-recovery)

* Removed section and all mention of Capped Collections
* Changed page title
* Reworked the introduction.

[Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=62dab8a1230aeb802efe53c8) with no errors.

### CODE REVIEW

File: source/reference/failure-recovery.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
 .. _c2c-failure-recovery:
 
-==========================
-Recovery From Sync Failure
-==========================
+=====================================
+Recovery Steps for Disaster Recovery
```
Comment (mdb-ashley) [inline source/reference/failure-recovery.txt:4]: [suggestion] This is really a tough page title. Perhaps, "Use `mongosync` for Disaster Recovery" although the question then becomes, how do you set up `mongosync` for disaster recovery in the first place?

File: source/reference/failure-recovery.txt [hunk 2]
```diff
@@ -12,22 +12,24 @@ Recovery From Sync Failure
    :depth: 2
    :class: singlecol
 
+Disaster Recovery is a use case of {+c2c-product-name+}.  Synchronization can
```
Comment (mdb-ashley) [inline source/reference/failure-recovery.txt:15]: [suggestion] I'd like to see a more descriptive intro sentence here. Perhaps setting up `mongosync` for disaster recovery is done the same way as any other connection? (Let's verify this.) If so, we could start with that. Something like, 

"You can use `mongosync` as a disaster recovery utility in the event of source cluster failure. [Connect your clusters](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/connecting/) and return to this page for instructions in the event you need disaster recovery."

File: source/reference/failure-recovery.txt [hunk 3]
```diff
@@ -12,22 +12,24 @@ Recovery From Sync Failure
...
+you must manually reset these collection characteristics on the destination
+cluster before you can use it as the primary cluster.
 
-Manually revert the following temporary changes:
+To prepare the destination cluster to replace the primary, make the following
+changes:
 
-- :ref:`Unique indexes set as non-unique <c2c-dr-unique>`
-- :ref:`TTL indexes with expireAfterSeconds set to MAX_INT
+- :ref:`Set Unique indexes as non-unique <c2c-dr-unique>`
```
Comment (mdb-ashley) [inline source/reference/failure-recovery.txt:28]: [praise] Great -- I like these changes a lot. Much clearer! 

[question] Should this be an ordered list?

**Review by mdb-ashley — APPROVED** [review]
LGTM with changes as requested. Thank for this speedy change, Kenneth!

**Review by alan-zheng-mdb — APPROVED** [review]

File: source/reference/failure-recovery.txt (hunk 1 above)
Comment (kennethdyer) [reply source/reference/failure-recovery.txt:4]: Fixed

File: source/reference/failure-recovery.txt (hunk 2 above)
Comment (kennethdyer) [reply source/reference/failure-recovery.txt:15]: Fixed

File: source/reference/failure-recovery.txt (hunk 3 above)
Comment (kennethdyer) [reply source/reference/failure-recovery.txt:28]: Fixed

File: source/reference/failure-recovery.txt [hunk 4]
```diff
@@ -12,22 +12,28 @@ Recovery From Sync Failure
...
 during synchronization. The original values are restored during the
-:ref:`commit process <c2c-api-commit>`. However, a permanent system
-failure could disrupt synchronization before the sync is committed. If
-the sync ends prematurely, you must manually reset the collection
-characteristics on the destination cluster.
+:ref:`commit process <c2c-api-commit>`. If synchronization ends prematurely,
+you must manually reset these collection characteristics on the destination
+cluster before you can use it as the primary cluster.
+
+To prepare the destination cluster to replace the primary, make the following
```
Comment (ldennis) [inline source/reference/failure-recovery.txt:28]: nit: replace the "source"?

File: source/reference/failure-recovery.txt [hunk 5]
```diff
@@ -12,22 +12,28 @@ Recovery From Sync Failure
...
+changes:
 
-Manually revert the following temporary changes:
+#. :ref:`Set Unique indexes as non-unique <c2c-dr-unique>`
+#. :ref:`Set TTL indexes with expireAfterSeconds to MAX_INT
+   <c2c-dr-ttl>`
+#. :ref:`Set Hidden indexes as non-hidden <c2c-dr-hidden>`
```
Comment (ldennis) [inline source/reference/failure-recovery.txt:34]: These should be the other way around.
During C2C, mongosync does the following as part of its replication machinery

- Set Unique indexes as non-unique
- Set TTL indexes with expireAfterSeconds to MAX_INT
- Set Hidden indexes as non-hidden

But users need to effectively undo this to get back their unique indexes/TTL indexes/hidden indexes.

**Review by ldennis — CHANGES_REQUESTED** [review]

File: source/reference/failure-recovery.txt (hunk 4 above)
Comment (kennethdyer) [reply source/reference/failure-recovery.txt:28]: Fixed

**Comment by alan-zheng-mdb** [comment]
nit - is this the right ticket to change the URL path from "failure recovery" to "disaster recovery"?

File: source/reference/failure-recovery.txt (hunk 5 above)
Comment (kennethdyer) [reply source/reference/failure-recovery.txt:34]: Fixed

**Review by ldennis — APPROVED** [review]
Great, I like the change. Thank you!

**Pull request merged**